source/includes/_admin.md

```diff
@@ -246,14 +295,28 @@ KillBillClient::Model::Admin.invalidates_cache(cache_name, options)
 ```python
 adminApi = killbill.api.AdminApi()
 
-adminApi.invalidates_cache(api_key, api_secret)
+adminApi.invalidates_cache()
```

Comment:
In some languages, we specify a cache name. Maybe we do it for all, for consistency?

Reply:
Noted, I have a list of small corrections across all the sections, I will address this at that time. Merging the PR for now.